src/structures/class/PropertyDeclarationStructure.ts

```diff
@@ -7,3 +7,7 @@ export interface PropertyDeclarationStructure
         JSDocableNodeStructure, ReadonlyableNodeStructure, InitializerExpressionableNodeStructure, DecoratableNodeStructure, AbstractableNodeStructure
 {
 }
+
+export interface PropertyDeclarationSpecificStructure
```

Comment:
Add this to the extends above to future proof it. Also, move the brace to the same line... I only did that above because there's a long extends clause.